packages/foreman/foreman/foreman.spec

```
@@ -991,6 +992,9 @@ exit 0
 %systemd_postun %{name}.socket
 
 %changelog
+* Mon Nov 21 2022 Quirin Pamp <pamp@atix.de> - 3.6.0-0.3.develop
+- Fix foreman-plugin macro
```

This could be a bit more descriptive:
```suggestion
- Ensure tmp directory exists in foreman_precompile_plugin
```

---

I have applied this, and also made the commit message more descriptive.